docs/modules/cloudmodule/controller.md

```diff
@@ -0,0 +1,55 @@
+# Edge Controller
+
+
+## Edge Controller Overview
+ controller(also known as edgecontroller) is the bridge between EdgeMaster(Kubernetes Api Server) and Edgecore
+
+   
+## Operations Performed By Edge Controller
+ 
+ The following are the functions performed by Edge controller :-
+  - Register and Start Sub Modules
+  - Downstream Controller-Sync add/update/delete event to edgeCore from EdgeMaster
+  - Upstream Controller -sync watch and Update status of resource and events(node,pod and configmap) to Edge master and also subscribe message from edgecore
+  - Controller Manager-Creates manager Interface and implements ConfigmapManager LocationCache and podManager
+### Register and Start Sub Modules
```

Comment:
This is not done by controller, please remove. This document should only have information related to controller